src/ConnectedMode/Binding/BindingProcessImpl.cs

```diff
@@ -221,7 +107,7 @@ public async Task<bool> SaveServerExclusionsAsync(CancellationToken cancellation
 
         public void SaveRuleConfiguration(CancellationToken cancellationToken)
         {
-            solutionBindingOperation.SaveRuleConfiguration(this.InternalState.BindingConfigs.Values, cancellationToken);
+            // TODO - remove. See #4662
```

Comment:
Second part of the two-step save. No longer required.
Will be done separately to avoid overcomplicating this PR.